22.mender-configure-module/docs.md

```diff
@@ -7,6 +7,44 @@ shortcode-core:
 github: false
 ---
 
+## mender-configure-module 1.0.3
+
+_Released 01.24.2022_
+
+### Statistics
+
+A total of 10 lines added, 7 removed (delta 3)
+
+| Developers with the most changesets | |
+|---|---|
+| Fabio Tranchitella | 2 (50.0%) |
+| Ole Petter Orhagen | 1 (25.0%) |
+| Lluis Campos | 1 (25.0%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Fabio Tranchitella | 7 (70.0%) |
+| Ole Petter Orhagen | 2 (20.0%) |
+| Lluis Campos | 1 (10.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 4 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 10 (100.0%) |
+
+| Employers with the most hackers (total 3) | |
+|---|---|
+| Northern.tech | 3 (100.0%) |
+
+
+### Changelogs
+
```

Comment:
Missing line `New changes in mender-connect since 1.0.2:` - it should have been added by the tool :thinking: